pyaedt/edb.py

```diff
@@ -1356,36 +1356,122 @@ def solve_siwave(self):
         return True
 
     @pyaedt_function_handler()
-    def add_design_variable(self, variable_name, variable_value):
+    def add_design_variable(self, variable_name, variable_value, is_parameter=False):
         """Add a design variable.
 
         Parameters
         ----------
         variable_name : str
-            Name of the variable.
+            Name of the variable, if it begins by a "$", it will be a "Project Variable".
         variable_value : str, float
             Value of the variable with units.
+        is_parameter : bool, optional
+            True to add a "Parameter Default", False for a "Local Variable"
 
         Returns
         -------
         tuple
             tuple containing the ``AddVariable`` result and variable server.
+
+        Examples
+        --------
+
+        >>> from pyaedt import Edb
+        >>> edb_app = Edb()
+        >>> boolean_1, ant_length = edb_app.add_design_variable("my_local_variable", "1cm")
+        >>> boolean_2, para_length = edb_app.change_design_variable_value("my_parameter", "1m", is_parameter=True
+        >>> boolean_3, project_length = edb_app.change_design_variable_value("$my_project_variable", "1m")
+
+
         """
-        is_parameter = True
         if "$" in variable_name:
-            var_server = self.db.GetVariableServer()
-            is_parameter = False
+            if variable_name.index("$") == 0:
+                var_server = self.db.GetVariableServer()
+                is_parameter = False
+                string_message = ["Creating Project Variable %s", "Project Variable %s exists. Using it."]
```

Comment:
```suggestion
                string_message = ["Creating project variable %s". "Project variable %s exists. Using it."]
```

Reply:
The second part is trying to say that the variable being created already exists so that the user can used it. string_message is a list so the two parts are used at different moments and conditions. string_message[0] is "creating project variable %s" and string_message[1] is "project variable $ exists. Using it" but I will convert it to "You can use it". They are two different messages.